test/e2e/fixture/app/consequences.go

```diff
@@ -112,7 +114,15 @@ func (c *Consequences) get() (*Application, error) {
 }
 
 func (c *Consequences) resource(kind, name, namespace string) ResourceStatus {
-	for _, r := range c.app().Status.Resources {
+	closer, client, err := fixture.ArgoCDClientset.NewApplicationClient()
```

Comment:
Instead of looking at the application directly, grab it from the server which populates the status using redis.